python/graphstorm/gconstruct/utils.py

```diff
@@ -1118,6 +1118,28 @@ def get_hard_edge_negs_feats(hard_edge_neg_ops):
 
     return hard_edge_neg_feats
 
+def get_gnid2pnid_map(ntype, node_mapping, gnid2pnid_mapping):
+    """ Get global nid to partitioned nid mapping.
+
+        Parameters
+        ----------
+        ntype: str
+            Path to the directory storing the partitioned graph.
+        node_mapping: dict
```

Comment:
Maybe have a more descriptive name here? The two mappings passed are the same mappings reversed right? `global_nid <-> partition_nid`

Reply:
I would say let us keep the naming for ``node_mapping``, because it is exactly the file name we are loading. But I will add one additional comment here to note what it is here.